packages/teams-js/src/internal/communication.ts

```diff
@@ -149,41 +149,16 @@ export function uninitializeCommunication(): void {
   CommunicationPrivate.portCallbacks = {};
 }
 
-/**
- * @hidden
- * Send a message to parent and then unwrap result. Uses nativeInterface on mobile to communicate with parent context
- * Additional apiVersionTag parameter is added, which provides the ability to send api version number to parent
- * for telemetry work. The code inside of this function will be used to replace sendAndUnwrap function
- * and this function will be removed when the project is completed.
- *
- * @internal
- * Limited to Microsoft-internal use
- */
-export function sendAndUnwrapWithVersion<T>(apiVersionTag: string, actionName: string, ...args: any[]): Promise<T> {
-  return sendMessageToParentAsyncWithVersion(apiVersionTag, actionName, args).then(([result]: [T]) => result);
-}
-
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
-export function sendAndUnwrap<T>(actionName: string, ...args: any[]): Promise<T> {
-  return sendMessageToParentAsync(actionName, args).then(([result]: [T]) => result);
+export function sendAndUnwrap<T>(apiVersionTag: string, actionName: string, ...args: any[]): Promise<T> {
+  return sendMessageToParentAsync(apiVersionTag, actionName, args).then(([result]: [T]) => result);
 }
 
-/**
- * @hidden
- * Send a message to parent and then handle status and reason. Uses nativeInterface on mobile to communicate with parent context
```

Comment:
Can you please keep these new comments that you wrote and stick them to the functions that they correspond to? It's great to have extra info here, and it would be a shame to lose this at the last second

Reply:
No problem. I'll keep it.